File: WebApplication/src/modules/application/application-five/models/dividend-report-result.dto.ts
import { NomenclatureDto } from "src/infrastructure/models/nomenclature.dto";
import { ApplicationFiveType } from "../enums/application-five-type.enum";

export class DividendReportResultDto {
  totalSum: number;
  amountRequestedCorrection: number;
  applicationFiveType: ApplicationFiveType;
  bank: NomenclatureDto;
  year: NomenclatureDto;
}

File: StudentCredit.Application/Applications/ApplicationsFive/Dtos/DividendReportResultDto.cs
using StudentCredit.Data.Applications.ApplicationFive.Enums;
using StudentCredit.Infrastructure.Helpers.Dtos;

namespace StudentCredit.Application.Applications.ApplicationsFive.Dtos
{
	public class DividendReportResultDto
	{
		public decimal TotalSum { get; set; }

		public decimal? AmountRequestedCorrection { get; set; }

		public ApplicationFiveType ApplicationFiveType { get; set; }

		public NomenclatureDto Bank { get; set; }

		public NomenclatureDto Year { get; set; }
	}
}
